docs/system/system-metrics.md

```diff
@@ -916,6 +917,44 @@ This metric is [recommended][MetricRecommended].
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `system.disk.limit`
+
+This metric is [recommended][MetricRecommended].
```

Comment:
Why is this recommended vs opt-in?

Reply:
Initially because the metric can not be derived from others, but seeing that all the "limit" metrics are opt-in I think it makes sense to change it here too: https://github.com/open-telemetry/semantic-conventions/pull/1356/commits/a6f673b1ef6b9b76771c75e998fa50b40ce5fe0a